packages/loader/container-loader/src/container.ts

```diff
@@ -593,8 +593,7 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
         this.mc = loggerToMonitoringContext(ChildLogger.create(this.subLogger, "Container"));
 
         const summarizeProtocolTree =
-            this.mc.config.getBoolean("Fluid.Container.summarizeProtocolTree")
-            ?? this.loader.services.options.summarizeProtocolTree;
+            this.mc.config.getBoolean("Fluid.Container.summarizeProtocolTree") ?? true;
```

Comment:
this shouldn't change. we don't want summarize protocol tree on by default yet, as we are still testing

Reply:
Oh, I though you said we want to make container upload default in the chat. Does that mean something else :)